External/Plugins/LintingHelper/Managers/LintingManager.cs

```diff
@@ -172,60 +161,46 @@ static void ApplyLint(string[] files, string language, List<LintingResult> resul
+                if (result.Length > 0)
+                {
+                    chars = $"chars {result.FirstChar}-{result.FirstChar + result.Length}";
+                }
+                else
                 {
-                    var start = doc.SciControl.PositionFromLine(result.Line - 1);
-                    var len = doc.SciControl.LineLength(result.Line - 1);
-                    start += result.FirstChar;
-                    if (result.Length > 0)
+                    var doc = DocumentManager.FindDocument(result.File);
+                    if (doc != null)
                     {
-                        len = result.Length;
+                        chars = $"chars {result.FirstChar}-{doc.SciControl.LineLength(result.Line - 1)}";
                     }
                     else
                     {
-                        len -= result.FirstChar;
-                    }
-
-                    var id = 0;
-                    int color = 0;
-                    var lang = PluginBase.MainForm.SciConfig.GetLanguage(language);
-                    switch (result.Severity)
-                    {
-                        case LintingSeverity.Error:
-                            color = lang.editorstyle.ErrorLineBack;
-                            id = 3;
-                            break;
-                        case LintingSeverity.Warning:
-                            color = lang.editorstyle.DebugLineBack;
-                            id = 4;
-                            break;
-                        case LintingSeverity.Info:
-                            color = lang.editorstyle.HighlightWordBackColor;
-                            id = 5;
-                            break;
+                        chars = $"char {result.FirstChar}";
                     }
-
-                    PluginBase.RunAsync(() =>
-                    {
-                        doc.SciControl.AddHighlight(id, (int)ScintillaNet.Enums.IndicatorStyle.Squiggle, color, start, len);
-                    });
                 }
+                string message = $"{result.File}:{result.Line}: {chars} : {result.Severity}: {result.Description}";
+                int state;
+                switch (result.Severity)
+                {
+                    case LintingSeverity.Info:
+                        state = (int) TraceType.Info;
+                        break;
+                    case LintingSeverity.Warning:
+                        state = (int) TraceType.Warning;
+                        break;
+                    case LintingSeverity.Error:
+                        state = (int) TraceType.Error;
+                        break;
+                    default:
+                        continue;
+                }
+                TraceManager.Add(message, state, TraceGroup);
             }
 
             PluginBase.RunAsync(() =>
             {
                 PluginBase.MainForm.CallCommand("PluginCommand", "ResultsPanel.ShowResults;" + TraceGroup);
```

Comment:
In my opinion, this line should be better deleted.
Currently, the linter results panel will jump up every time linting is done (file save, file open), even if the user closed the panel, which should then be accessible using the view menu.

If this line is deleted, on the very first creation of the linter results panel, the panel will be visible. Then, it won't be "forced shown". In other words, it doesn't jump up, nor shows when manually closed by the user.